Tests/fixtures/templateSpecificLocale.twig

```diff
@@ -0,0 +1 @@
+{{ yesterday|ago(null, 'fr') }}
```

Comment:
```suggestion
{{ yesterday|ago(null, 'fr') }}

```

Reply:
By adding a new line in the template, it changes the string returned on the test. Here is my outup:

```
PHPUnit 8.5.13 by Sebastian Bergmann and contributors.

Testing Project Test Suite
.....F                                                              6 / 6 (100%)

Time: 305 ms, Memory: 24.00 MB

There was 1 failure:

1) Knp\Bundle\TimeBundle\Tests\IntegrationTest::testLocalTranslation
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'il y a 1 jour'
+'il y a 1 jour
+'
```

Do we still want that? Also, template.twig in that folder only has 1 line. I think for consistency, it would be better it stay that way.

Reply:
agreed. disregard this suggestion. thanks!